assets/css/settings.scss

```diff
@@ -558,14 +559,94 @@
 	}
 }
 
+.sensei-learners-wrap .search-box {
+	margin-left: 10px;
+}
+
+.sensei-student-bulk-actions {
+	&__filters {
+		display: flex;
+		flex-direction: row;
+	}
+
+	&__filters > div {
+		margin-right: 10px;
+	}
+
+	@media screen and (max-width: 1400px) {
+		&__filters {
+			display: flex;
+			flex-direction: column-reverse;
+		}
+	}
+
+	@media screen and (max-width: 1400px) and (min-width: 1100px) {
+		&__container {
+			width: 60%;
+		}
+	}
+
+	@media screen and (max-width: 1400px) {
+		&__wrapper {
+			height: 30px;
+			margin-bottom: 10px
+		}
+	}
+}
+
+.sensei-student-bulk-actions__wrapper .sensei-student-bulk-actions__filters .sensei-student-bulk-actions__placeholder-dropdown {
```

Comment:
I'm wondering if it's necessary for this CSS selector to be so specific? The problem with making selectors this specific is that it's a lot harder for a third party developer to override it if they want to. 

From what I can tell, there are 2 different elements that have the `sensei-student-bulk-actions__placeholder-dropdown` class, but I _think_ only [this one](https://github.com/Automattic/sensei/pull/4997/files#diff-195ee038793e99cd0caabbe4025ae7060394007908b0783ff2c1faa54e932e2aR316) actually needs it, and [this one](https://github.com/Automattic/sensei/pull/4997/files#diff-195ee038793e99cd0caabbe4025ae7060394007908b0783ff2c1faa54e932e2aR279) does not (from what I can tell, we're not styling that one at all)?

Assuming the above is true, if we remove the class from the one that doesn't need it, then we should just be able to use `.sensei-student-bulk-actions__placeholder-dropdown` here and also [here](https://github.com/Automattic/sensei/pull/4997/files#diff-5c437b51ca7067e0a3307a080cfa9c2315d7fe7d62744683bd4972bbd62869deR626).

Taking things a bit further, do we need to introduce a new class at all, or could we use `sensei-bulk-action-select` to style it?

Reply:
So, the reason it is so specific is that it is made to be replaced with a select2 selector.  
And the reason why it is made to resemble select2 selector is to avoid the **glitching** when it is replaced.
So with this class before they are replaced with select2 they look almost identical to the selector they will be replaced with. I took this screenshot with select2 disabled: 
<img width="393" alt="image" src="https://user-images.githubusercontent.com/7208249/162910864-3db88d4f-3d21-400e-8ed5-b37a9e8426da.png">

If it si not so specific I need to use !important and I was trying to avoid it as much as possible.
They have different classes that are used by select2 so using sensei-bulk-action-select will not cover the other one also like this if we add new selector and we want it to be select2 we can just add this dropdown class to avoid gliching